src/server/api/controllers/waypoints.controller.js

```diff
@@ -23,7 +23,7 @@ const getWaypointsById = async id => {
 // Method for creating a waypoints
 
 const createWaypoints = async ({ body }) => {
-  const { latitude, longitude, suggested_route_id } = body;
+  const { latitude, longitude, suggested_route_id, speed, rpm } = body;
```

Comment:
Sequence_id is missing

Reply:
Actually not. Since this method is for creating a single "standalone" waypoint, not a set of waypoints as part of a route, it will not make sense to declare a sequence_id.

When we are creating a suggested_route, however, we should be able to add a series of waypoints and set their sequence ID.